iset.mm

```diff
@@ -147339,6 +147339,448 @@ since the target of the homomorphism (operator ` O ` in our model) need
+      LKZMFJKZFLKZMENKFNKATUBUAUCABCDEFGHIOABCDEFGHIPQERFRS $.
+  $}
+
+  ${
+    $d x y K $.  $d x y L $.
+    ablprop.b $e |- ( Base ` K ) = ( Base ` L ) $.
+    ablprop.p $e |- ( +g ` K ) = ( +g ` L ) $.
+    $( If two structures have the same group components (properties), one is an
+       Abelian group iff the other one is.  (Contributed by NM,
+       11-Oct-2013.) $)
+    ablprop $p |- ( K e. Abel <-> L e. Abel ) $=
+      ( vx vy cabl wcel wb wtru cbs cfv eqidd wceq a1i cv cplusg co wa oveqi
+      ablpropd mptru ) AGHBGHIJEFAKLZABJUCMUCBKLNJCOEPZFPZAQLZRUDUEBQLZRNJUDUCH
+      UEUCHSSUFUGUDUEDTOUAUB $.
+  $}
+
+  ${
+    $d x y B $.  $d x y G $.  $d x y ph $.
+    iscmnd.b $e |- ( ph -> B = ( Base ` G ) ) $.
+    iscmnd.p $e |- ( ph -> .+ = ( +g ` G ) ) $.
+    iscmnd.g $e |- ( ph -> G e. Mnd ) $.
+    iscmnd.c $e |- ( ( ph /\ x e. B /\ y e. B ) -> ( x .+ y ) = ( y .+ x ) ) $.
+    $( Properties that determine a commutative monoid.  (Contributed by Mario
+       Carneiro, 7-Jan-2015.) $)
+    iscmnd $p |- ( ph -> G e. CMnd ) $=
+      ( cmnd wcel cv cfv co wceq wral oveqd raleqbidv eqid cplusg cbs wa 3expib
+      ccmn ralrimivv eqeq12d anbi2d mpbi2and iscmn sylibr ) AFKLZBMZCMZFUANZOZU
+      NUMUOOZPZCFUBNZQZBUSQZUCZFUELAULUMUNEOZUNUMEOZPZCDQZBDQZVBIAVEBCDDAUMDLUN
+      DLVEJUDUFAVGVAULAVFUTBDUSGAVEURCDUSGAVCUPVDUQAEUOUMUNHRAEUOUNUMHRUGSSUHUI
+      BCUSUOFUSTUOTUJUK $.
+  $}
+
+  ${
+    $d x y B $.  $d x y G $.  $d x y ph $.
+    isabld.b $e |- ( ph -> B = ( Base ` G ) ) $.
+    isabld.p $e |- ( ph -> .+ = ( +g ` G ) ) $.
+    isabld.g $e |- ( ph -> G e. Grp ) $.
+    isabld.c $e |- ( ( ph /\ x e. B /\ y e. B ) -> ( x .+ y ) = ( y .+ x ) ) $.
+    $( Properties that determine an Abelian group.  (Contributed by NM,
+       6-Aug-2013.) $)
+    isabld $p |- ( ph -> G e. Abel ) $=
+      ( cgrp wcel ccmn cabl grpmndd iscmnd isabl sylanbrc ) AFKLFMLFNLIABCDEFGH
+      AFIOJPFQR $.
+  $}
+
+  ${
+    $d x y B $.  $d x y G $.
+    isabli.g $e |- G e. Grp $.
+    isabli.b $e |- B = ( Base ` G ) $.
+    isabli.p $e |- .+ = ( +g ` G ) $.
+    isabli.c $e |- ( ( x e. B /\ y e. B ) -> ( x .+ y ) = ( y .+ x ) ) $.
+    $( Properties that determine an Abelian group.  (Contributed by NM,
+       4-Sep-2011.) $)
+    isabli $p |- G e. Abel $=
+      ( cabl wcel cgrp cv co wceq wral rgen2 isabl2 mpbir2an ) EJKELKAMZBMZDNUA
+      TDNOZBCPACPFUBABCCIQABCDEGHRS $.
+  $}
+
+  ${
+    $d x y B $.  $d x y G $.  $d x y .+ $.  $d x y W $.  $d x y X $.
+    $d x y Y $.  $d x y Z $.
+    $( A commutative monoid is a monoid.  (Contributed by Mario Carneiro,
+       6-Jan-2015.) $)
+    cmnmnd $p |- ( G e. CMnd -> G e. Mnd ) $=
+      ( vx vy ccmn wcel cmnd cv cplusg cfv co wceq cbs wral eqid iscmn simplbi
+      ) ADEAFEBGZCGZAHIZJRQSJKCALIZMBTMBCTSATNSNOP $.
+
+    ablcom.b $e |- B = ( Base ` G ) $.
+    ablcom.p $e |- .+ = ( +g ` G ) $.
+    $( A commutative monoid is commutative.  (Contributed by Mario Carneiro,
```

Comment:
Better: The (group) operation of a commutative monoid is commutative.

Reply:
What's the benefit of the extra words?  I'm not sure how to misread the original.

Also, I thought "group operation" with respect to a monoid which might not be a group was controversial, so perhaps it is better to just sidestep that issue?

Reply:
To be more precise, not the structure, but its operation is commutative (see also comment for ~ablcom: "An Abelian group operation is commutative."). 
It is, however, customary to omit the "operation" (see, for example, https://mathworld.wolfram.com/CommutativeMonoid.html: "A monoid that is commutative ...").
Therefore, this remark is nonblocking.

Reply:
Strictly speaking, commutativity is a property of the base set as well as the operation (especially in the presence of https://us.metamath.org/mpeuni/df-ress.html which truncates the base set without truncating the operation).

I believe a real world example is quaternion multiplication, which is not commutative, but becomes commutative when restricted to complex numbers.